app/views/layouts/_footer.html.haml

```diff
@@ -2,28 +2,8 @@
   .container
     .row
       .col-md-4#about-growstuff
-        %ul
-          %li= link_to t('about'), "http://wiki.growstuff.org/index.php/About%20Growstuff"
-          %li= link_to t('our_values'), "http://wiki.growstuff.org/index.php/Values"
-          %li= link_to t('open_source'), "https://github.com/Growstuff/growstuff"
-          %li= link_to t('growstuff_team'), "http://wiki.growstuff.org/index.php/Team"
-          %li= link_to t('get_involved'), "http://wiki.growstuff.org/index.php/Get_involved"
+        != cms_snippet_content(:footer1)
```

Comment:
I specifically didn't want "about, legal, contact" because I might want 
to change what's where, in future.  I wouldn't mind doing the column 
names, but I don't feel strongly about it.  It would need to be 
footer_middle_column or something though, which starts to get wordy.

A.


On 9/02/2015 7:00 pm, Taylor Griffin wrote:
>
> In app/views/layouts/_footer.html.haml 
> <https://github.com/Growstuff/growstuff/pull/694#discussion_r24313107>:
>
> > @@ -2,28 +2,8 @@
> >    .container
> >      .row
> >        .col-md-4#about-growstuff
> > -        %ul
> > -          %li= link_to t('about'), "http://wiki.growstuff.org/index.php/About%20Growstuff"
> > -          %li= link_to t('our_values'), "http://wiki.growstuff.org/index.php/Values"
> > -          %li= link_to t('open_source'), "https://github.com/Growstuff/growstuff"
> > -          %li= link_to t('growstuff_team'), "http://wiki.growstuff.org/index.php/Team"
> > -          %li= link_to t('get_involved'), "http://wiki.growstuff.org/index.php/Get_involved"
> > +        != cms_snippet_content(:footer1)
>
> This is probably not a big deal (and I don't know anything about 
> Comfortable Mexican Sofa), but would it make sense for these to have 
> more meaningful names other than |:footer1|, |:footer2|, and 
> |:footer3|? Maybe |:about|, |:legal|, and |:contact|? Or even 
> |:left_column|, |:middle_column|, and |:right_column|?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Growstuff/growstuff/pull/694/files#r24313107>.
>

-- 
Alex "Skud" Bayley
skud@growstuff.org
http://growstuff.org/